frontend/src/components/Figure.astro

```diff
@@ -0,0 +1,26 @@
+---
+import ImageLink from "./ImageLink.astro";
+
+export interface Props {
```

Comment:
I'm all for adding in TS defs. I know Val has complained in the past that he doesn't like having part of the project with TS and part without. At the time, I'm not sure what he was talking about unless he was complaining about my custom Apollo code having Doc Blocks.

Reply:
TS here is "for free" (Astro configures it for the project). It makes sense to utilize it for at least the stateless components, because it would lint incorrect usage. For the actual Apostrophe data, we need a typescript support and a way to bring it to the astro project - a long road ahead. It makes no sense to "describe" the schema s TS types.